recipes/pyqt5/bld.bat

```diff
@@ -0,0 +1,11 @@
+::CALL "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x64
+
+%PYTHON% configure.py ^
+        --spec=win32-msvc2010 ^
```

Comment:
Sorry, we haven't had time to work on the Windows part :-) It'll come later this week.